packages/wrangler/src/index.tsx

```diff
@@ -390,7 +390,7 @@ export async function main(argv: string[]): Promise<void> {
 
   // dev
   yargs.command(
-    "dev <filename>",
+    "dev <filename> [-- build]",
```

Comment:
Could we add other, non-build, flags in the future that would be compatible with this? e.g. `--dry-run`

Reply:
Yeah for sure. We already support `--port`, we can add more. 

Honestly, I'm not fully sold on this blank `--` thing yet, could be confusing. Might be worth just having explicit `--build <command>`.

Reply:
` -- ` makes the `build` argument become positional. Arguably in this case we should support this positional being used without the `--`. E.g. `wrangler dev npm build`, rather than `wrangler dev -- npm build`. At least this is my understanding of the convention around `--` (i.e. it indicates the end of options and anything after is positional).

So with that in mind, I would prefer to have build as a clear option: `wrangler dev --build "npm build"`.

Reply:
as discussed, I removed the `--` option altogether.